student.rb

```diff
@@ -4,9 +4,14 @@ class Student < Person
   def initialize(age, classroom, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission)
     @classroom = classroom
+    add_classroom(@classroom)
   end
 
   def play_hooky
     '¯\(ツ)/¯'
   end
+
+  def add_classroom(classroom)
+    classroom.students.push(self) unless classroom.students.include?(self)
+  end
 end
```

Comment:
- [ ] You might have forgotten to add the attribute reader for a classroom. Kindly include it ```attr_reader :classroom```.